src/App.vue

```diff
@@ -696,13 +698,6 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
         throw error // go to catch()
       })
 
-      // get Keycloak roles
-      const keycloakRoles = await this.loadKeycloakRoles().catch(error => {
-        console.log('Keycloak roles error =', error) // eslint-disable-line no-console
-        this.accountAuthorizationDialog = true
-        throw error // go to catch()
-      })
```

Comment:
In the old code, we decoded the KC roles and we fetch Authorizations (from Auth API). Both return a similar list.

Fetching the authorizations from Auth API is better because "row level permissions" can be added.

So, I stripped out the KC code.

Reply:
I did this in a separate PR: https://github.com/bcgov/business-create-ui/pull/785

This conversation is out of date - resolving it.